view/main_content_panel.py

```diff
@@ -262,11 +268,60 @@ def refresh_cache_calculations(self) -> None:
             self.local_mods[uuid]["data_source"] = "local"
 
         # One working Dictionary for ALL mods
-        all_mods = merge_mod_data(self.expansions, self.local_mods, self.workshop_mods)
+        all_mods = merge_mod_data(
+            self.expansions, self.local_mods, self.workshop_mods)
         logger.info(
             f"Combined {len(self.expansions)} expansions, {len(self.local_mods)} local mods, and {len(self.workshop_mods)}. Total elements to get dependencies for: {len(all_mods)}"
         )
 
+        # Check mods verified working with newer game-versions than defined
+        # https://steamcommunity.com/sharedfiles/filedetails/?id=2599504692
+        package_id_to_uuid = {}
+        for mod_uuid, modmetadata in all_mods.items():
+            package_id_to_uuid[modmetadata["packageId"]] = mod_uuid
+        if "mlie.noversionwarning" in package_id_to_uuid.keys():
+            no_version_mod = all_mods[package_id_to_uuid["mlie.noversionwarning"]]
```

Comment:
Minor: may want to extract this to a function in the module

Reply:
That code was cloned from another part of the mod so it would probably be a good idea